twilio/rest/accounts/v1/docs/AccountsV1AuthTokenPromotion.md

```diff
@@ -1,13 +1,209 @@
 # AccountsV1AuthTokenPromotion
 
 ## Properties
-Name | Type | Description | Notes
------------- | ------------- | ------------- | -------------
-**AccountSid** | Pointer to **string** | The SID of the Account that the secondary Auth Token was created for |
-**AuthToken** | Pointer to **string** | The promoted Auth Token |
-**DateCreated** | Pointer to [**time.Time**](time.Time.md) | The ISO 8601 formatted date and time in UTC when the resource was created |
-**DateUpdated** | Pointer to [**time.Time**](time.Time.md) | The ISO 8601 formatted date and time in UTC when the resource was last updated |
-**Url** | Pointer to **string** | The URI for this resource, relative to `https://accounts.twilio.com` |
+
+Name | Type | Description
+------------ | ------------- | -------------
+**AccountSid** | Pointer to **NullableString** | The SID of the Account that the secondary Auth Token was created for | [optional] 
+**AuthToken** | Pointer to **NullableString** | The promoted Auth Token | [optional] 
+**DateCreated** | Pointer to **NullableTime** | The ISO 8601 formatted date and time in UTC when the resource was created | [optional] 
+**DateUpdated** | Pointer to **NullableTime** | The ISO 8601 formatted date and time in UTC when the resource was last updated | [optional] 
+**Url** | Pointer to **NullableString** | The URI for this resource, relative to &#x60;https://accounts.twilio.com&#x60; | [optional] 
```

Comment:
The mustache templates default to HTML escaping, which causes the URLs to be rendered as `https://accounts.twilio.com&#x60;` in all of the readme files. Can you wrap this description in [triple mustaches to fix these links](http://mustache.github.io/mustache.5.html)?